crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -329,7 +338,9 @@ impl Client {
                     thumbnail_info
                 });
                 let content = assign!(ImageMessageEventContent::encrypted(body.to_owned(), file), {
-                    info: Some(Box::new(info))
+                    info: Some(Box::new(info)),
+                    formatted: formatted,
+                    filename: filename
```

Comment:
nit: I think you can shorthand those two:

```suggestion
                    formatted,
                    filename
```